multi-transfer-esdt/src/token_mapping.rs

```diff
@@ -0,0 +1,22 @@
+multiversx_sc::imports!();
+
+#[multiversx_sc::module]
+pub trait TokenMappingModule {
+    #[only_owner]
+    #[endpoint(setSovToMxTokenId)]
+    fn set_sov_to_mx_token_id(&self, sov_token_id: TokenIdentifier, mx_token_id: TokenIdentifier) {
+        require!(
+            sov_token_id.is_valid_esdt_identifier() && mx_token_id.is_valid_esdt_identifier(),
+            "Invalid token IDs"
+        );
+
+        self.sovereign_to_multiversx_token_id(&sov_token_id)
```

Comment:
Do you use the same storage for `multivers_to_sovereign`? If so, it's hard to get  `sov_token_id` from `mx_token_id`

Reply:
We don't need the reverse storage. The bridge on the other side will handle this.